cayleypy/graphs_lib.py

```diff
@@ -48,13 +48,18 @@ def _create_coxeter_generators(n: int) -> list[list[int]]:
     return gens
 
 
-def prepare_graph(name, n=0) -> CayleyGraph:
+def prepare_graph(name, n=0, **kwargs) -> CayleyGraph:
     """Returns pre-defined Cayley or Schreier coset graph.
 
     Supported graphs:
       * "all_transpositions" - Cayley graph for S_n (n>=2), generated by all n(n-1)/2 transpositions.
       * "pancake" - Cayley graph for S_n (n>=2), generated by reverses of all prefixes. It has n-1 generators denoted
           R1,R2..R(n-1), where Ri is reverse of elements 0,1..i. See https://en.wikipedia.org/wiki/Pancake_graph.
+      * "cubic_pancake" - Cayley graph for S_n (n>=2), generated by set of 3 prefix reversal generators.
+        Sets definitions are:
+          S=1 => {R_n, R_n-1, R_2},    S=2 => {R_n, R_n-1, R_3},    S=3 => {R_n, R_n-1, R_n-2},
```

Comment:
Are these standard notations, do they come from somewhere? Can you add a reference (website or paper)?

Also I would list them one on each line, and it shouldn't end with a comma.